jvm-libs/linea/blob-compressor/src/main/kotlin/net/consensys/linea/blob/GoNativeBlobCompressor.kt

```diff
@@ -1,8 +1,8 @@
 package net.consensys.linea.blob
 
+import build.linea.jvm.ResourcesUtil.copyResourceToTmpDir
 import com.sun.jna.Library
 import com.sun.jna.Native
-import net.consensys.jvm.ResourcesUtil.copyResourceToTmpDir
 
 interface GoNativeBlobCompressor {
```

Comment:
Can we name this as `BlobCompressor` to align with `BlobDecompressor`?

Reply:
That would require a deeper refactor to be properly done.  
`BlobDecompressor` is a high level abstraction, whereas `GoNativeBlobCompressor` is mapping/binding to the Go interface. Therefore the names are correct. We just don't have that abstraction for the compressor here and it's out of scope of this PR.

Reply:
If that's the case, why do we name the new interface as `BlobDecompressor` instead of `GoNativeBlobDecompressor`?

Reply:
Read the code carefully, as I explained `BlobDecompressor` is a higher level abstraction. 

Ther is an other interface `GoNativeBlobDecompressorJnaBinding` which is the equivalent of `GoNativeBlobCompressor `. The problem is the Compressor poor decoupling, the de decompressor